fair/.htaccess

```diff
@@ -4,7 +4,9 @@ RewriteEngine on
 
 RewriteRule ^$ https://www.go-fair.org/ [R=302,L]
 
-RewriteRule ^metrics/(.*)$ https://github.com/FAIRMetrics/Metrics/#$1 [R=302,NE,L]
+RewriteRule ^maturity_indicator/$ https://github.com/FAIRMetrics/Metrics [R=302,L]
```

Comment:
Might want to use `^maturity_indicator/?$` if you want the trailing `/` to be optional.

Reply:
Right, thanks for pointing that out! I might do that for the next version of the rewrite rules.